app/ticket/page.tsx

```diff
@@ -0,0 +1,76 @@
+'use client';
+import { useState } from 'react';
+import CheckOut from './components/CheckOut';
+import Summary from './components/Summary';
+import { AnimatePresence, motion } from 'framer-motion';
+import TicketComp from './components/TicketComp';
+
+export default function Ticket() {
+  const [selectedPackage, setSelectedPackage] = useState({
+    key: '',
+    day: '',
+    activityType: '',
+    price: 0,
+  });
+  const [view, setView] = useState('checkout');
+  const [formData, setFormData] = useState({
+    fullName: '',
+    email: '',
+  });
+
+  const renderFn = () => {
+    switch (view) {
+      case 'checkout':
+        return (
+          <CheckOut
+            selectedPackage={selectedPackage}
+            setSelectedPackage={setSelectedPackage}
+            formData={formData}
+            setFormData={setFormData}
+            setView={setView}
+          />
+        );
+      case 'summary':
+        return <Summary setView={setView} selectedPackage={selectedPackage} />;
+      case 'success':
+        return <TicketComp selectedPackage={selectedPackage} />;
+      default:
+        return (
+          <CheckOut
+            selectedPackage={selectedPackage}
+            setSelectedPackage={setSelectedPackage}
+            formData={formData}
+            setFormData={setFormData}
+            setView={setView}
+          />
+        );
+    }
+  };
+
+  return (
+    <>
+      <div
+        className="h-screen w-full flex items-center justify-center px-5"
+        style={{
+          backgroundImage: "url('/ticket_bg.png')",
+          backgroundRepeat: 'no-repeat',
+          backgroundPosition: 'center',
+          backgroundSize: 'cover',
+        }}
```

Comment:
There are two different background images, one for desktop and the other for mobile

Deskop background-image
<img width="827" height="464" alt="image" src="https://github.com/user-attachments/assets/32de1ed3-0496-48ea-b5a0-bc5dff12d681" />

Mobile 
<img width="224" height="473" alt="image" src="https://github.com/user-attachments/assets/be89be51-2116-4328-93f9-0401fa93966d" />

Switch the background image to that of the mobile on mobile devices, I think max-width:768px